docs/.gitignore

```diff
@@ -0,0 +1,2 @@
+
+_build/*
```

Comment:
The default `sncli/.gitignore` already includes `build`. So leaving the Makefile at default and using `-BUILDDIR = build` would do the same thing as adding this file and then editing the Makefile.

Reply:
The leading underscore makes its purpose more apparent. Also the `build` in the root gitignore is there for an unrelated purpose.